src/dynarec/la64/dynarec_la64_00.c

```diff
@@ -2384,7 +2384,7 @@ uintptr_t dynarec64_00(dynarec_la64_t* dyn, uintptr_t addr, uintptr_t ip, int ni
                     if (dyn->last_ip && (addr - dyn->last_ip < 0x800)) {
                         ADDI_D(x2, xRIP, addr - dyn->last_ip);
                     } else {
-                        TABLE64(x2, addr);
+                        MOV64x(x2, addr);
```

Comment:
just to note: this will need a table64 when relocation is done on this arg (unless you add relocation on MOV64x type, wich is doable, but anoying to do), because addr is relative the mmap, wich can be moved on another load.

Reply:
yeah yeah I’m aware, this is to align the implementation before finishing dynacache so I won’t miss anything.